images/identity.go

```diff
@@ -0,0 +1,56 @@
+package images
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type IdentityImage struct {
+	KeyUID       string
+	Name         string
+	Payload      []byte
+	Width        int
+	Height       int
+	FileSize     int
+	ResizeTarget int
+}
+
+func (i IdentityImage) GetType() (ImageType, error) {
+	it := GetType(i.Payload)
+	if it == UNKNOWN {
+		return it, errors.New("unsupported file type")
+	}
+
+	return it, nil
+}
+
+func (i IdentityImage) GetDataURI() (string, error) {
+	return GetPayloadDataURI(i.Payload)
+}
+
+func (i IdentityImage) MarshalJSON() ([]byte, error) {
+	uri, err := i.GetDataURI()
+	if err != nil {
+		return nil, err
+	}
+
+	temp := struct {
+		KeyUID       string `json:"key_uid"`
```

Comment:
we generally use `camelCase` for json, though we are not super consistent, as sometimes we use `kebab-case` (clojurism). And sometimes `snake_case` but that's mainly protobuf default serialization.
Ideally eventually we go for `camelCase` everywhere